chord_metadata_service/restapi/api_renderers.py

```diff
@@ -188,7 +189,7 @@ def render(self, data, media_type=None, renderer_context=None):
                 'id': biosample['id'],
                 'description': biosample.get('description', 'NA'),
                 'sampled_tissue': biosample.get('sampled_tissue', {}).get('label', 'NA'),
-                'individual_age_at_collection': biosample.get('individual_age_at_collection', {}).get('age', 'NA'),
+                'individual_age_at_collection': render_age(biosample, "time_of_collection"),
```

Comment:
this should probably be renamed `time_of_collection` to match phenopackets v2. other than this, it looks good!